RULES.md

```diff
@@ -85,3 +81,7 @@ This table lists rules proposed by the community but refused or/and deleted in e
 |----------|----------|------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | CRDOM203 | HTML     | HTML page must contain a doctype tag     | The difference in performance is negligible, this rule is more related to the user experience.                               | [Github discussion with sources](https://github.com/green-code-initiative/ecoCode/issues/103)                                                                                    |
 | CRPYT    | Python   | Use numpy array instead of standard list | The use of numpy library to perform array manipulation is more energy efficient than the use of the standard list functions. | [Github discussion with measures](https://github.com/green-code-initiative/ecoCode/issues/132)                                                                                   |
+| EC4      | Java                | Avoid using global variables                       | Global variables do not exist in Java.                                                                   | [Github discussion with sources](https://github.com/green-code-initiative/ecoCode/issues/233)                                                                                                                                                                |
+| EC53     | Java                | Using arrays in foreach loops                      | No good arguments and not enough green measures.                                                         | [Github discussion with sources](https://github.com/green-code-initiative/ecoCode/issues/240)                                                                                                                                                                |
```

Comment:
my bad, this EC53 rule has to be deleted because deprectaed but move this line in array "Rules to be reworked / measured / clarified" instead in order to work on this rule with measurement later.

Reply:
ok